pinot-core/src/main/java/org/apache/pinot/core/data/partition/PartitionFunctionFactory.java

```diff
@@ -29,7 +29,7 @@
 public class PartitionFunctionFactory {
   // Enum for various partition functions to be added.
   public enum PartitionFunctionType {
-    Modulo, Murmur, ByteArray;
+    Modulo, Murmur, ByteArray, Simple;
```

Comment:
Should we name it something more explicit, like HashCode?

Reply:
Changed to `HashCodePartitionFunction`.